src/i965_gpe_utils.c

```diff
@@ -1406,6 +1406,27 @@ i965_object_surface_to_2d_gpe_resource(struct i965_gpe_resource *res,
 }
 
 void
+i965_object_surface_to_2d_gpe_resource_with_align(struct i965_gpe_resource *res,
+                                                  struct object_surface *obj_surface)
+{
+    unsigned int swizzle;
```

Comment:
i965_object_surface_to_2d_gpe_resource() also will be used by other function, such as VPP. it will cause the issue to change the value in the function i965_object_surface_to_2d_gpe_resource(). that is the reason why add the version with aligned